woocommerce/woocommerce
# Add isBlock to ToggleGroupControl instances which unifies their styling

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Some of the ToggleGroupControl didn't occupy the full Inspector Control width or had no border around. This PR unifies them by adding `isBlock` prop which applies the same styles.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create new post
2. Insert the blocks below and verify the controls using ToggleGroupControl (as per screenshots) are full width:

Block | Before | After
-- | -- | --
Product Image (insert Single Product block and look for Product Image inside) | <img width="283" alt="image" src="https://github.com/user-attachments/assets/b42b9004-a2ce-4019-9773-7f11742d063e"> | <img width="283" alt="image" src="https://github.com/user-attachments/assets/00f9ae75-aabe-410c-ba23-f2d646b8e655">
Active Filters | <img width="279" alt="image" src="https://github.com/user-attachments/assets/9ada6477-5e39-4243-8b2d-bf5062028f0f"> | <img width="285" alt="image" src="https://github.com/user-attachments/assets/ae770ae3-8501-4257-87db-f6e491792088">
Customer Account | <img width="282" alt="image" src="https://github.com/user-attachments/assets/a1138475-9d71-4c63-8709-1b63633d4c03"> | <img width="285" alt="image" src="https://github.com/user-attachments/assets/925bf220-1837-48c9-80da-f344a82b51c5">
Featured Product (only on product with main image!) | <img width="283" alt="image" src="https://github.com/user-attachments/assets/1d86c836-d881-4c8f-b9b7-58871ee71761"> | <img width="279" alt="image" src="https://github.com/user-attachments/assets/2d1ce98c-ef94-468d-b774-ff9965e1d596">
Filter by Attribute | <img width="278" alt="image" src="https://github.com/user-attachments/assets/6f88e31d-d413-43b8-aa49-288796652e0a"> | <img width="282" alt="image" src="https://github.com/user-attachments/assets/1d39c9b6-60b8-41ed-9f80-3ebd7312a220">
Filter by Price |<img width="282" alt="image" src="https://github.com/user-attachments/assets/3a031825-4a79-466a-9928-228e4ed382c8"> | <img width="284" alt="image" src="https://github.com/user-attachments/assets/7023f40d-e4bd-406d-8294-f223eb88a0e4">
Filter by Rating | <img width="279" alt="image" src="https://github.com/user-attachments/assets/088c7cf6-a96e-46b8-9658-224ccda889f7"> | <img width="283" alt="image" src="https://github.com/user-attachments/assets/c432db46-4565-402c-9a76-71550091342a">
Filter by Stock Status | <img width="283" alt="image" src="https://github.com/user-attachments/assets/b0a05443-93b8-41c7-8518-628905a6cf8d"> | <img width="282" alt="image" src="https://github.com/user-attachments/assets/bfbdd9d9-0332-4966-b74d-68514e13fbe4">
Reviews | <img width="284" alt="image" src="https://github.com/user-attachments/assets/ab471f44-d7ee-41d0-864e-cb61f00d39d4"> | <img width="285" alt="image" src="https://github.com/user-attachments/assets/1f422e21-2947-4215-8f47-635dae404d3e">

3. Go to Single Product template
4. Replace Product Image Gallery with Product Gallery (Beta)
5. Focus on Product Gallery (Beta) and browse inner blocks:

Block | Before | After
-- | -- | --
Next/Previous Buttons | <img width="284" alt="image" src="https://github.com/user-attachments/assets/3236ad72-7ebc-4877-bb72-eca7874dffcb"> | <img width="281" alt="image" src="https://github.com/user-attachments/assets/e393ad51-b54f-43c3-be93-6aa7ce51888f">
Pager | <img width="279" alt="image" src="https://github.com/user-attachments/assets/a68e8ae8-8219-4420-af27-c29e96185596"> | <img width="279" alt="image" src="https://github.com/user-attachments/assets/77b4dcd8-6302-49ae-b957-52980eb31a61">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
